### PR TITLE
feat: add width/height to /screenshot POST request

### PR DESCRIPTION
**Pull request by luroc**
As requested in #37 

This is probably a quite naive implementation, as I have next to no experience with Rust, but it works for me so far.

### CODE REVIEW

File: src/routes/screenshot.rs [hunk 1]
```diff
@@ -51,6 +65,8 @@ pub async fn screenshot(
         _ => Error::WebsiteError,
     })?;
 
+    let width = payload.width;
+    let height = payload.height;
```
Comment (devtomio) [inline src/routes/screenshot.rs:69]:
```suggestion
```

File: src/routes/screenshot.rs [hunk 2]
```diff
@@ -66,7 +82,7 @@ pub async fn screenshot(
     let client = &data.browser;
 
     client.goto(url.as_str()).await.expect("Failed navigating to site");
-    client.set_window_size(1980, 1080).await.expect("Failed setting window size");
+    client.set_window_size(width, height).await.expect("Failed setting window size");
```
Comment (devtomio) [inline src/routes/screenshot.rs:85]:
```suggestion
    client.set_window_size(payload.width, payload.height).await.expect("Failed setting window size");
```

File: src/routes/screenshot.rs (hunk 1 above)
Comment (devtomio) [inline src/routes/screenshot.rs:69]:
```suggestion
```

File: src/routes/screenshot.rs (hunk 2 above)
Comment (devtomio) [reply src/routes/screenshot.rs:85]: just set it on the function

**Review by devtomio — CHANGES_REQUESTED** [review]
a few nits

**Comment by luroc** [comment]
Looks good to me (but what do I know ;) )

Mind also that your default was 1980x1080 and I changed that to 1920x1080, in case you missed that and the 1980 had a specific reason.

**Review by devtomio — APPROVED** [review]

**Pull request merged**